arcoirsl_webstore/database/migrations/2023_01_30_163105_create_products_table.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('products', function (Blueprint $table) {
+            $table->id();
+            $table->string('SKU');
+            $table->string('brand_name');
+            $table->string('item_name');
+            $table->string('external_product_id');
+            $table->string('external_product_id_type');
+            $table->int('unit_count');
+            $table->string('SKU');
```

Comment:
Use **FOREIGN KEY**

https://laravel.com/docs/9.x/migrations#foreign-key-constraints

there is some example:

`
$table->foreign('brand_id')->references('id')->on('brands');
`
`
$table->foreign('info_id')->references('id')->on('product_infos');
`